README.md

```diff
@@ -138,32 +149,48 @@ pg_dump -h localhost -p 5432 -U <MIGRATION_USER> -d <DATABASE_NAME> -f /data/dum
 ```
 
 ### 3. Gjenopprett dumpet data
+Når data er dumpet til pod kan det gjenopprettes i ønsket database. Først må gjeldende `cloud_sql_proxy` fjernes, og ny proxy settes opp mot
+databasen det skal importeres til som angitt i steg 1.
 
-Når data er dumpet til pod kan det gjenopprettes i ønsket database.
-Eksempel
+Videre kan import kjøres på følgende måte:
+```shell
+psql -h localhost -p 5432 -U <MIGRATION_USER> <DATABASE_NAME> -f /data/dump.sql
 ```
-cloud_sql_proxy -enable_iam_login -instances=etterlatte-prod-207c:europe-north1:etterlatte-sakogbehandlinger=tcp:5432 &
-psql -h localhost -p 5432 -U migration-user@etterlatte-dev-9b0b.iam sakogbehandlinger -f /data/test.sql
+
+## Cleanup
+
+Når du er ferdig med migrering kan du kjøre:
+
+```shell
```

Comment:
Sletter dette alle ressursene egentlig? Kan ikke huske det gjorde det for oss

Reply:
Usikker. Det lå der fra før. Kanskje vi burde teste det før vi kjører neste migrering?